src/DTF/Libraries/WindowsInstaller.Package/InstallPackage.cs

```diff
@@ -191,6 +191,7 @@ public void ExtractFiles(ICollection<string> fileKeys)
     {
         this.ProcessFilesByMediaDisk(fileKeys,
             new ProcessFilesOnOneMediaDiskHandler(this.ExtractFilesOnOneMediaDisk));
+        this.ExtractFilesInBinaryTable(fileKeys);
```

Comment:
Can't make this change in v3.x since it's a significant behavior change for all users of this method in DTF. I think the best approach for v3.x is to add a switch to melt to specify the binary output directory. If not specified, don't extract binaries. And that could be all internal to melt rather than a DTF change. Another approach would be to do kinda what dark does and extract files and binaries to their own subdirectories. That's also a behavior change but I could be convinced it's OK...